src/models/MediaImage.ts

```diff
@@ -46,6 +46,48 @@ export class MediaImageL {
     get app(): Maybe<CushyAppL> {return this.draft?.app} // prettier-ignore
     get script(): Maybe<CushyScriptL> {return this.app?.script } // prettier-ignore
 
+    copyToClipboard = async () => {
+        const canvas = document.createElement('canvas')
+        const ctx = canvas.getContext('2d')
+
+        const img = new Image()
```

Comment:
Just in case, there are already some functions to help loading images here: `src/state/createHTMLImage_fromURL.ts`

```ts
export const createHTMLImage_fromURL = (
    /** the same `src` value you would use in an <img /> html node */
    src: string,
): Promise<HTMLImageElement> => {
    return new Promise((yes, no) => {
        const img = new Image()
        img.loading = 'eager'
        img.onload = () => yes(img)
        img.onerror = no
        img.src = src
    })
}

export const createHTMLImage_fromURLNoAwait = (
    /** the same `src` value you would use in an <img /> html node */
    src: string,
): HTMLImageElement => {
    const img = new Image()
    img.loading = 'eager'
    // img.onload = () => yes(img)
    // img.onerror = no
    img.src = src
    return img
}
```